aries-site/src/examples/components/tooltip/ToolTipExitExample.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button, Tip, Text, Heading } from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const ToolTipExitExample = () => {
+  return (
+      <Box align="center" justify="center" fill>
+        <Box gap="xsmall" round="xsmall">
+          <Box justify="between" direction="row">
+            <Heading level={4} size="small" margin="none">
```

Comment:
Let's use text / bold / large here since we can't anticipate if h4 will be the proper semantic heading level